src/screens/ProfileScreen/AvgCycleScreen.js

```diff
@@ -11,7 +11,7 @@ import {
 } from 'react-native';
 import { AntDesign } from '@expo/vector-icons';
 import SmoothPicker from 'react-native-smooth-picker';
-
+import i18n from '../../i18n';
```

Comment:
use `@/i18n` conventions to import the module. Please check another import statement in other files.

Reply:
Okay